keras_cv/models/mlp_mixer.py

```diff
@@ -57,12 +57,15 @@
     For transfer learning use cases, make sure to read the [guide to transfer
         learning & fine-tuning](https://keras.io/guides/transfer_learning/).
     Args:
-        include_rescaling: whether or not to Rescale the inputs.If set to True,
-            inputs will be passed through a `Rescaling(1/255.0)` layer.
-        include_top: whether to include the fully-connected layer at the top of the
-            network.  If provided, num_classes must be provided.
-        num_classes: optional number of classes to classify images into, only to be
-            specified if `include_top` is True.
+
```

Comment:
hey @soma2000-lang there are a lot of spurious changes here (newlines in docstrings, changes from snake case and TitleCase to camelCase, etc).

Would you mind reverting these spurious changes and re-opening?